Guidelines.md

```diff
@@ -1096,7 +1096,9 @@ If the delta link is no longer valid, the service MUST respond with _410 Gone_.
 
 ## 11. JSON standardizations
 ### 11.1. JSON formatting standardization for primitive types
-Primitive values MUST be serialized to JSON following the rules of [RFC4627][rfc-4627].
+Primitive values MUST be serialized to JSON following the rules of [RFC4627][rfc-4627]. 
+
+**Important note for 64bit integers:** JavaScript will silently truncate integers larger than `Number.MAX_SIZE_INTEGER` (2^53-1). If the service is expected to return larger integer values, consider returning the value as a string. 
```

Comment:
In JavaScript at least, it's Number.MAX_SAFE_INTEGER.

Worth noting that there is also Number.MIN_SAFE_INTEGER and numbers below -2^53+1 are not interoperable?

Reply:
"Consider" is somewhat weaker than I was hoping 😄 We should at least "strongly consider" not exposing users to silent data loss I think!

Reply:
Strongly consider it is. And, apparently my JavaScript skills have deteriorated. Updated to MAX/MIN_SAFE...